authors/apps/authentication/tests/base_class.py

```diff
@@ -13,6 +13,8 @@ def setUp(self):
 
         self.url_register = reverse('authentication:register')
         self.url_login = reverse('authentication:login')
+        self.url_user_detail = reverse('authentication:retrieve_update')
+        self.expired_token = 'eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJpZCI6MSwiZXhwIjoxNTQ4OTU2MzY5fQ.Eu-0EFROqMrPcWEkcgRJYG0zzK_-IuhODAe8v69rd5o'
```

Comment:
I would suggest this moved to the `test_data` file, since it looks like it will be a permanent value.

Reply:
Alright, I am going to move that up.
Great feedback @Karuhanga.